betterforms/multiform.py

```diff
@@ -16,6 +16,14 @@
 from django.utils.six.moves import reduce
 
 
+class classproperty(property):
```

Comment:
Shouldn't this be written from scratch? So that you don't have to combine it with `@classmethod`.

It looks very simple, something like:

```python
class classproperty(object):
  def __init__(self, func):
     self.func = func

  def __get__(self, cls, owner):
     ...
```